allauth/socialaccount/providers/authentiq/provider.py

```diff
@@ -58,14 +59,16 @@ class AuthentiqProvider(OAuth2Provider):
     account_class = AuthentiqAccount
 
     def get_scope(self, request):
+        adapter = get_adapter()
         scope = set(super(AuthentiqProvider, self).get_scope(request))
         scope.add("openid")
 
         if Scope.EMAIL in scope:
             modifiers = ""
             if app_settings.EMAIL_REQUIRED:
                 modifiers += "r"
-            if app_settings.EMAIL_VERIFICATION:
+            #TODO: How to get user email here for get_email_verification_method ?
```

Comment:
How can I get user's email in this method to pass it to get_email_verification_method ?

Reply:
This method is called to get a list of scopes to ask permission for as part of the OAuth handshake. So, at this point in time you do not know what user is going to be authenticated, yet.

Reply:
So, in this case login parameter in get_email_verification_method can be None ?